hangupsbot/plugins/slackrtm.py

```diff
@@ -1168,22 +1179,27 @@ def handle_reply(self, reply):
                     else:
                         # we should not upload the images, so we have to send the url instead
                         response += msg.file_attachment
-                self.sending += 1
                 self.loop.call_soon_threadsafe(asyncio.async,
-                    self.bot.coro_send_message(sync.hangoutid, response, context={'slack': True})
+                    self.bot.coro_send_message(
+                        sync.hangoutid, response,
+                        context = {
+                            'base': {
+                                'tags': ['slack', 'relay'],
+                                'source': 'slackrtm',
+                                'importance': 50
+                            },
+                            'reprocessor': self.bot.call_shared("reprocessor.attach_reprocessor",
+                                self._repeater_cleaner, return_as_dict=True)
+                        }
+                    )
                 )
 
     @asyncio.coroutine
     def handle_ho_message(self, event):
+        if "_slackrtm_no_repeat" in dir(event) and event._slackrtm_no_repeat:
+            return
+
         for sync in self.get_syncs(hangoutid=event.conv_id):
-            if self.sending and ': ' in event.text:
-                # this hangout message originated in slack
-                self.sending -= 1
-                command = event.text.split(': ')[1]
-                event.text = command
-                logger.debug('attempting to execute %s', command)
-                yield from self.bot._handlers.handle_command(event)
-                return
```

Comment:
in retrospect, i misunderstood the nature of #796 due to inexperience with the slackrtm plugin and spectacularly uninformative issue descriptions - it appears there are two issues associated with slackrtm, and only one of them is being reported: 
* /bot` commands issued in slack are not being run - re: #796 
* `@hobot` commands are no longer recognised when entered on the slack side either

neither of these features are dear to me right now (and I'm not even sure they are gonna make the cut to v3 chatbridges) so I'm simply putting this here as an FYI